addons/resource.language.en_gb/resources/strings.po

```diff
@@ -8640,8 +8655,10 @@ msgctxt "#19056"
 msgid "New timer"
 msgstr ""
 
+#. Label for a context menu entry to open timer settings dialog
+#: xbmc/pvr/windows/GUIWindowPVRTimers.cpp
 msgctxt "#19057"
-msgid "Edit timer"
+msgid "Settings"
```

Comment:
it was removed for music/pictures as context item to go straight to system settings